### ISSUE TITLE
"No parser and no filepath given"

### ISSUE THREAD

**Issue by wirekang** [imjuni/ctix#42]
.ctirc:
```json
{
  "project": "./tsconfig.json",
  "exportFilename": "index.ts",
  "useSemicolon": false,
  "useTimestamp": false,
  "useComment": false,
  "quote": "'",
  "overwrite": true,
  "keepFileExt": false,
  "skipEmptyDir": true,
  "output": "./",
  "useRootDir": false,
  "includeBackup": false
}
```

Command:
`npx ctix create`

Output:
```No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.```

**Comment by imjuni** [comment]
Hi @wirekang 

ctix works fine on my project what [jin-frame](https://github.com/imjuni/jin-frame) and [erdia](https://github.com/imjuni/erdia).

Can you create example repository for solve this problem?

**Comment by Chanki-Min** [comment]
Hello @imjuni

Looks like this is prettier of ctix's problem
prettier needs `parser` field to run. and if not, `No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.` warning will be thrown

And looking at [prettier useage in ctix's code](https://github.dev/imjuni/ctix/blob/9bd0760ca16dc904dc7ad80632770b7fa94d09ff/src/writes/prettierApply.ts#L9), looks like ctix using host project's prettier config (`.prettirc`)

So, If host project (who uses ctix) doesn't have `.prettirc` there are no `parser` config given to ctix's prettier instance.

ref) https://github.com/vuejs/vue-cli/issues/3356#issuecomment-953449132

By the way, I'm very grateful to use this library. I wish this could helps

**Comment by Chanki-Min** [comment]
I deleted my project's `.prettierrc` and ran ctix than same warning rises :)

**Comment by imjuni** [comment]
If you set the `.editorconfig` when you are not using the prettier, it will occur when the pretter works.

This issue has been fixed in v1.3.3, and even if `.editorconfig` is set, the prettier will not work unless parser is set to `typecript` in the prettier option.

@wirekang @Chanki-Min Thank you for your help fix this problem.
